### PR TITLE
[DOCU-2943] Add portal auth API docs to Konnect docs

### PR DESCRIPTION
**Pull request by cloudjumpercat**
Signed-off-by: Diana <75819066+cloudjumpercat@users.noreply.github.com>

### Summary

- Added portal auth API spec to Konnect.
- Added links to the portal auth documentation to the Konnect docs.
- modified RBAC guide contribution to remove repeats of the spec. (AG)
- updated URL to new version of the spec (AG)
- aligned to GA in JIRA

### Reason

DOCU-2787 (old) DOCU-2943 (actual)
### Testing

Check the links in the API section of the Konnect docs.

### CODE REVIEW

**Review by Guaris — APPROVED** [review]
lgtm

Hayden should merge this.

**Comment by acgoldsmith** [comment]
FLAGGING DEPENDENCY that API must be available at https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/edcaaf42-777b-47f9-aa80-797327a4b8b2/ or else 404s will occur.

**Comment by acgoldsmith** [comment]
v2 of the spec is confirmed at https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696 (thanks, @Guaris ) and all links changed to that destination.

**Comment by acgoldsmith** [comment]
@sunnytoolsmiths, this is ready for GA release. Recommend enabling the service in devportal first to ensure the spec is live at  at https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696 ahead of merging this PR.

**Comment by acgoldsmith** [comment]
@sunnytoolsmiths, this is ready for GA release. Spec is already public.

File: app/konnect/api/portal-auth/portal-rbac-guide.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  --url https://<region>.api.konghq.com/konnect-api/api/portals/<portal-id> \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "rbac_enabled": true
+  }'
+```
+
+{:.note}
+> The `portal-id` can be found in the {{site.konnect_short_name}} Portal within the **Developers** tab.
```
Comment (nateslo) [inline app/konnect/api/portal-auth/portal-rbac-guide.md:30]: The current wording is a little confusing to me. 

image


`{{site.konnect_short_name}}` is replaced with `Konnect Portal`, but I think it should just be `Konnect`. Also, the portal id has been added to all subsections of the Dev Portal Section of Konnect, not just the Developers Tab (I believe this is recent enhancement)


I think it should be something like: 
```
The `portal-id` can be found in the upper right corner of the **Dev Portal** section in Konnect.
```




image

Comment (nateslo) [inline app/konnect/api/portal-auth/portal-rbac-guide.md:30]:
```suggestion
> The `portal-id` can be found in {{site.konnect_short_name}} within the **Dev Portal** section. 
```

**Pull request merged**